tests/renderStatsCard.test.js

```diff
@@ -245,7 +245,7 @@ describe("Test renderStatsCard", () => {
       document.querySelector(
         'g[transform="translate(0, 25)"]>.stagger>.stat.bold',
       ).textContent,
-    ).toBe("总提交 (2020):");
+    ).toBe("总提交 (2021):");
```

Comment:
I'm not sure whether we should check this `year`'s number? 🤔 ...

Anyway, it is the reason why test failed, I fixed.

Reply:
or we need to write code to check it dynamically? 🤔